ui/app/AppLayouts/Wallet/views/TransactionSettings.qml

```diff
@@ -250,43 +310,53 @@ Rectangle {
-                            const rawEnteredValue = root.fnGasToRaw(customBaseFeeInput.text)
+                            const rawEnteredValue = root.fnGasToRaw(customBaseFeeOrGasPriceInput.text)
                             return rawEnteredValue.cmp(increasedCurrentValue) === 1
                         }
 
                         Layout.preferredWidth: parent.width
                         Layout.topMargin: 20
-                        label: qsTr("Max base fee")
+                        label: !root.fromChainEIP1559Compliant? qsTr("Gas price") : qsTr("Max base fee")
                         labelIcon: "info"
                         labelIconColor: Theme.palette.baseColor1
                         labelIconClickable: true
-                        bottomLabelMessageLeftCmp.color: customBaseFeeInput.displayLowBaseFeeWarning
-                                                         || customBaseFeeInput.displayHighBaseFeeWarning?
+                        bottomLabelMessageLeftCmp.color: customBaseFeeOrGasPriceInput.displayLowBaseFeeOrGasPriceWarning
+                                                         || customBaseFeeOrGasPriceInput.displayHighBaseFeeOrGasPriceWarning?
                                                              Theme.palette.miscColor6
                                                            : Theme.palette.baseColor1
-                        bottomLabelMessageLeftCmp.text: customBaseFeeInput.displayLowBaseFeeWarning?
-                                                            qsTr("Lower than necessary (current %1)").arg(Utils.nativeTokenRawToGas(root.currentBaseFee))
-                                                          : customBaseFeeInput.displayHighBaseFeeWarning?
-                                                                qsTr("Higher than necessary (current %1)").arg(Utils.nativeTokenRawToGas(root.currentBaseFee))
-                                                              : qsTr("Current: %1 GWEI").arg(Utils.nativeTokenRawToGas(root.currentBaseFee))
+                        bottomLabelMessageLeftCmp.text: {
+                            const baseFeeOrGasPrice = !root.fromChainEIP1559Compliant?
+                                                        root.currentGasPrice
+                                                      : root.currentBaseFee
+
+                            return customBaseFeeOrGasPriceInput.displayLowBaseFeeOrGasPriceWarning?
+                                        qsTr("Lower than necessary (current %1)").arg(root.fnRawToGas(baseFeeOrGasPrice).toFixed())
+                                      : customBaseFeeOrGasPriceInput.displayHighBaseFeeOrGasPriceWarning?
+                                            qsTr("Higher than necessary (current %1)").arg(root.fnRawToGas(baseFeeOrGasPrice).toFixed())
+                                          : qsTr("Current: %1 GWEI").arg(root.fnRawToGas(baseFeeOrGasPrice).toFixed())
+                        }
```

Comment:
I assume these functions above will not correctly display the floating point numbers in the user's locale?

Reply:
@caybro I haven't checked, but it's now exactly the same as it was. What do you think it should be like?

Reply:
Just because those are all integers, but guess it should work without it as well.